### PR TITLE
Flutter create project for build dashboard v2

### PR DESCRIPTION
**Pull request by CaseyHillers**
All of this is from the output when running flutter create with version 1.9

Updated the README to include a link to the design doc.

### CODE REVIEW

**Comment by jonahwilliams** [comment]
The source files in this repository will need a copyright header like the other projects. These should be something like:

```
// Copyright (c) 2019 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
```

**Comment by jonahwilliams** [comment]
cc @dnfield , how do we hook up the unit tests from this project to CI?

File: app_flutter/test/widget_test.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
```
Comment (jonahwilliams) [inline app_flutter/test/widget_test.dart]: The test file needs one too

File: app_flutter/web/index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (jonahwilliams) [inline app_flutter/web/index.html]: You need one here in an html comment, see https://github.com/flutter/flutter/blob/master/examples/hello_world/web/index.html#L1

File: app_flutter/test/widget_test.dart (hunk 1 above)
Comment (CaseyHillers) [reply app_flutter/test/widget_test.dart]: Added! Thank you!

File: app_flutter/web/index.html (hunk 2 above)
Comment (CaseyHillers) [reply app_flutter/web/index.html]: Added! Thank you!

**Comment by dnfield** [comment]
We should add another task for testing Flutter - with a Flutter enabled image.  Right now, the .cirrus.yml image we use only has Dart.

Alternatively, we could just switch entirely to the Flutter image and use dart from that image, which is probably just easier to think about anyway.

**Pull request merged**